package.json

```diff
@@ -13,7 +13,10 @@
     "prisma:generate": "npx prisma generate",
```

Comment:
Everything in this file is still in double quotes. We might want to update this if we are changing everything else.

Reply:
This is a json file so everything should be in ""